.travis.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         - $SCCACHE --version
       script:
         - cargo fmt -- --check
+        - cargo build
```

Comment:
Don't think this is needed. Build failure is because of the thing I mentioned in the call that ethcontract doesn't see the address. But Felix said he would push a fix for that.